tests/D2L.CodeStyle.Analysis.UnitTests/D2L.CodeStyle.Analysis.UnitTests.csproj

```diff
@@ -41,12 +41,40 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CodeAnalysis, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.CodeAnalysis.Common.1.3.2\lib\net45\Microsoft.CodeAnalysis.dll</HintPath>
+    <Reference Include="Castle.Core, Version=3.3.0.0, Culture=neutral, PublicKeyToken=407dd0808d44fbdc, processorArchitecture=MSIL">

```

Comment:
(Moq appears to have added it in 4.5.7.  Possibly when it stopped using ILMerge?)